src/azure-cli/azure/cli/command_modules/resource/tests/latest/management_group_level_parameters.json

```diff
@@ -3,7 +3,7 @@
   "contentVersion": "1.0.0.0",
   "parameters": {
     "nestedsubId": {
-      "value": "0b1f6471-1bf0-4dda-aec3-cb9272f09590"
+      "value": "a1bfa635-f2bf-42f1-86b5-848c674fc321az"
```

Comment:
Please do not change the `nestedsubId` of this test. If you don't have the permission to execute this test, I can rerecord yaml files instead of you

Reply:
I do not have the permissions to execute this test, please do.